src/consensus/pbft/libbyz/Replica.cpp

```diff
@@ -215,15 +218,13 @@ void Replica::receive_message(const uint8_t* data, uint32_t size)
     LOG_DEBUG
       << "Received message will not be processed, size exceeds message limits: "
```

Comment:
maybe alter the log message to log the size but not to say that the message will not be processed.